src/api/render-node.test.ts

```diff
@@ -79,4 +81,35 @@ test.serial('[integration] getNodeManagement responds according to Ethereum and
   t.assert(res.orchestrator);
 
   scope.done();
+
+  // mock docker hub state with a few new versions
+  const scope2 = nockDockerHub(
```

Comment:
spelling mistake? lol :) no, the library that mocks HTTP is called "nock", so Amir called it `nockDockerHub` because it uses `nock` to mock it. 

https://www.npmjs.com/package/nock